conf/application.conf

```diff
@@ -91,8 +91,8 @@ sitnet.integration.courseUnitInfo.active = true
 # "default" is used if no other match is found. You can remove the "default" key if you don't want a default URL at all.
 sitnet.integration.courseUnitInfo.url = {
   default = "http://base.uni.org?courseCode=${course_code}"
-  a.uni.org = "http://a.uni.org?courseCode=${course_code}"
-  b.uni.org = "http://b.uni.org?courseCode=${course_code}"
+  orgA = "http://a.uni.org?courseCode=${course_code}"
```

Comment:
These configs that contain dots are problematic to resolve. Therefore rephrasing the example.